.circleci/continue_config.yml

```diff
@@ -4,6 +4,21 @@ orbs:
   node: circleci/node@4.7.0
   ruby: circleci/ruby@1.1.4
 
+aliases:
+  - &androidWorkspace
+      ~/apps/android/LiveVideoApp
+  - &android-gradle-cache-key
+    key: jars-{{ checksum "apps/android/LiveVideoApp/build.gradle" }}-{{ checksum "apps/android/LiveVideoApp/gradle/wrapper/gradle-wrapper.properties" }}
+  - &android-restore-cache-gradle
+    <<: *android-gradle-cache-key
+    name: Restore Gradle Cache
+  - &android-save-cache-gradle
+    <<: *android-gradle-cache-key
+    name: Save Gradle Cache
+    paths:
+      - ~/apps/android/LiveVideoApp/.gradle/caches
+      - ~/apps/android/LiveVideoApp/.gradle/wrapper
```

Comment:
In this case, I implemented the `aliases` because at when the job was running the command to `build all APKs` and running the command for Unit Tests, was trying to write new files on the folder but it wasn't allow due to writing permissions on that folder. So, I noticed using aliases for `save cache` and `restore cache` was the solution for it.